rewindable-index/src/RewindableIndex/Storable.hs

```diff
@@ -231,10 +250,42 @@ getEvents
   -> StorableMonad h [StorableEvent h]
 getEvents s = do
   memoryEs <- getMemoryEvents (s ^. storage)
-              & V.freeze <&> V.toList
+              & V.freeze <&> foldEvents . V.toList
   diskEs   <- getStoredEvents (s ^. handle)
   pure $ diskEs ++ memoryEs
 
+{- This function is used to add a checkpoint to the in-memory part of event indexers.
+   You can use checkpoints to add synthetic events to the memory buffer. Filling the
+   memory buffer will flush the real events to disk faster than not using checkpoints.
```

Comment:
Do you mean instead

```suggestion
   memory buffer will flush the real events to disk faster than using checkpoints.
```

Reply:
No, but I think I could use better english :)